secret-handshake.json

```diff
@@ -0,0 +1,142 @@
+    ],
+    "commands": {
+        "description": ["Create a handshake for a number"],
+        "cases": [
+            {
+                "description": "shake for 1",
+                "input": 1,
+                "expected": ["wink"]
+            },
+            {
+                "description": "shake for 10",
+                "input": 2,
+                "expected": ["double blink"]
+            },
+            {
+                "description": "shake for 100",
+                "input": 4,
+                "expected": ["close your eyes"]
+            },
+            {
+                "description": "shake for 1000",
+                "input": 8,
+                "expected": ["jump"]
+            },
+            {
+                "description": "shake for 11",
+                "input": 3,
+                "expected": ["wink", "double blink"]
+            },
+            {
+                "description": "shake for 11 reversed",
+                "input": 19,
+                "expected": ["double blink", "wink"]
+            },
+            {
+                "description": "shake for all",
+                "input": 16,
+                "expected": ["wink", "double blink", "close your eyes", "jump"]
+            },
+            {
+                "description": "shake for all reversed",
+                "input": 31,
+                "expected": ["jump", "close your eyes", "double blink", "wink"]
+            },
+            {
+                "description": "shake for binary",
+                "input": "101",
+                "expected": ["wink", "close your eyes"]
+            },
+            {
+                "description": "shake for reversed binary",
+                "input": "10110",
+                "expected": ["close your eyes", "double blink"]
+            },
+            {
+                "description": "invalid binary",
+                "input": "121",
+                "expected": []
+            },
+            {
+                "description": "shake for 0",
+                "input": 0,
+                "expected": []
+            },
+            {
+                "description": "shake for negative",
+                "input": -1,
+                "expected": []
+            },
+            {
+                "description": "too big",
+                "input": 32,
+                "expected": []
+            },
+            {
+                "description": "invalid string",
+                "input": "piggies",
+                "expected": []
+            },
+            {
+                "description": "invalid with numbers",
+                "input": "1piggies",
+                "expected": []
+            }
+        ]
+    },
+    "decode": {
+        "description": ["Decode a handshake to binary"],
+        "cases": [
+            {
+                "description": "decode big",
+                "actions": ["close your eyes", "jump"],
+                "expected": "1100"
```

Comment:
should the output be in string form (representing a binary number), or integer form?